rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.raw_api.mustache

```diff
@@ -0,0 +1,169 @@
+# This file was automatically generated by BuildFileGenerator
+
+# This is an API workspace, having public visibility by default makes perfect sense.
+package(default_visibility = ["//visibility:public"])
+
+##############################################################################
+# Common
+##############################################################################
+load("@com_google_googleapis_imports//:imports.bzl", "proto_library_with_info")
+
+proto_library(
+    name = "{{name}}_proto",
+    srcs = [
+        {{proto_srcs}}
+    ],
+    deps = [
+        {{proto_deps}}
+    ],
+)
+
+proto_library_with_info(
```

Comment:
Is it necessary to import and set up this target when it isn't used in this "raw" build file? If not, let's remove it and the load statement for it.

Reply:
I think you are right, it is not needed and is even harmful. Removed those.